docs/userguide/how_to_configure_simulations.rst

```diff
@@ -0,0 +1,331 @@
+    envelope :math:`1`.
+
+First, construct the components of the model:
+
+.. jupyter-execute::
+
+    import numpy as np
+    from qiskit.quantum_info import Operator
+    from qiskit_dynamics import Solver, Signal
+
+    dim = 5
+
+    v = 5.
+    anharm = -0.33
+    r = 0.1
+
+    a = np.diag(np.sqrt(np.arange(1, dim)), 1)
+    adag = np.diag(np.sqrt(np.arange(1, dim)), -1)
+    N = np.diag(np.arange(dim))
+
+    # static part
+    static_hamiltonian = 2 * np.pi * v * N + np.pi * anharm * N * (N - np.eye(dim))
+    # drive term
+    drive_hamiltonian = 2 * np.pi * r * (a + adag)
+    # drive signal
+    drive_signal = Signal(1., carrier_freq=v)
+
+    # total simulation time
+    T = 1. / r
+
+Construct a ``Solver`` for the model as stated, without entering a rotating frame, and solve,
+timing the solver.
+
+.. jupyter-execute::
+
+    solver = Solver(
+        static_hamiltonian=static_hamiltonian,
+        hamiltonian_operators=[drive_hamiltonian],
+        hamiltonian_signals=[drive_signal]
+    )
+
+    y0 = np.eye(dim, dtype=complex)
+    %time results = solver.solve(t_span=[0., T], y0=y0, atol=1e-10, rtol=1e-10)
+
+Next, define a ``Solver`` in the rotating frame of the static
+Hamiltonian by setting the ``rotating_frame`` kwarg, and solve, again timing the solver.
+
+.. jupyter-execute::
+
+    rf_solver = Solver(
+        static_hamiltonian=static_hamiltonian,
+        hamiltonian_operators=[drive_hamiltonian],
+        hamiltonian_signals=[drive_signal],
+        rotating_frame=static_hamiltonian
+    )
+
+    y0 = np.eye(dim, dtype=complex)
+    %time rf_results = rf_solver.solve(t_span=[0., T], y0=y0, atol=1e-10, rtol=1e-10)
+
+Observe that despite the two simulation problems being mathematically equivalent, it takes
+less time to solve in the rotating frame.
+
+Next, verify that the results are numerically equivalent. This requires
+transforming the results to a common frame, which may be done via utility
+functions in the ``RotatingFrame`` instance stored within ``Solver.model.rotating_frame``.
+
+To compare the results, we use the fidelity function for unitary matrices:
+
+.. math:: f(U, V) = \frac{|Tr(U^\dagger V)|^2}{d^2},
+
+where :math:`d` is the dimension. A value of :math:`1` indicates equality of the unitaries.
+
+.. jupyter-execute::
+
+    def fidelity(U, V):
+        # the fidelity function
+        inner_product = (U.conj() * V).sum()
+        return (np.abs(inner_product) / dim) ** 2
+
+    U = results.y[-1]
+    # transform the results of the solver in the rotating frame out of the rotating frame
+    U_rf = rf_solver.model.rotating_frame.state_out_of_frame(T, rf_results.y[-1])
+
+    fidelity(U, U_rf)
+
+Based on the fidelity, we see that the two simulations are numerically equivalent
+with reasonable accuracy based on our specified tolerances.
+
+The discrepancy in solving times can be understood by examining the number of
+RHS evaluations when solving in each instance. The number of RHS evaluations for
```

Comment:
Maybe define RHS here in case a new user isn't familiar with the terminology. Although maybe everyone who is using this package already has a strong understanding?

Reply:
Changed this to: `right-hand side (RHS)`, and added: "when solving the differential equation". I think this is a good addition, and _should_ be sufficient, all the key words are there. Also yeah, if a user is at the point of wanting to configure simulations they should know what these words mean at this point anyway.

Reply:
Agreed, great